CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Added
+* Support for accessing arbitrary MSBuild properties, see [Readme section](#access-msbuild-properties) ([#210])
```

Comment:
I think the link should be `[…](README.md#access-msbuild-properties)`

Reply:
woops - fixed 814b691